entity/BUILD.gn

```diff
@@ -14,6 +14,8 @@ impeller_shaders("entity_shaders") {
     "shaders/solid_fill.vert",
     "shaders/solid_stroke.frag",
     "shaders/solid_stroke.vert",
+    "shaders/texture_blend_gaussian_blur.frag",
```

Comment:
Just gaussian_blur perhaps? Not sure if the additional verbosity adds anything.

Reply:
Done.